src/components/LandingPage.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { theme } from '../styling/theme';
+
+import AccentedLink from './molecules/atoms/AccentedLink';
+import OutlinedButton from './molecules/atoms/OutlinedButton';
+
+interface ILandingPage {
+  className?: string;
+}
+
+const BrandTitle = styled.h1`
+  font-family: "Montserrat", sans-serif;
+  text-transform: uppercase;
+  font-size: 4em;
+  font-weight: 300;
+  color: ${theme.contrast};
```

Comment:
Would be nice to have properties like font-familiy, size etc. in the theme as well for reusability and consistent design throughout the site.

Reply:
Font-family is defined in the global styling file, here I'm just overriding as I want the brand to pop a bit more. As for the font-size, I believe it is so individual for each component that it's no use in putting it in a global file.